.github/workflows/test_frameworks.yml

```diff
@@ -94,6 +94,39 @@ jobs:
         # django.tests.requests module interferes with requests library patching in the tracer -> disable requests patch
         run: DD_TRACE_REQUESTS_ENABLED=0 ddtrace-run tests/runtests.py
 
+  graphne-testsuite-3_0:
+    runs-on: ubuntu-latest
+    env:
+      DD_PROFILING_ENABLED: true
+      DD_TESTING_RAISE: true
+    defaults:
+      run:
+        working-directory: graphene
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v3
+        with:
+          repository: graphql-python/graphene
+          ref: v3.0.0
+          path: graphene
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install ddtrace
+        run: pip install ../ddtrace
+      - name: Install graphene
+        run: pip install -e "../graphene[test]"
+      - name: "Upgrade pytest"
+        # pytest<6.2.5 raises TypeError: required field "lineno" missing from alias on python 3.10
```

Comment:
and pytest >= 7 is broken?

Reply:
I remember seeing somewhere that it was not supported but I believe I mixed up graphene's requirements with  graphql-core. I remove the  pytest<7 restriction

Reply:
oh so we don't need to install pytest explicitly since it's done above in `pip install -e "../graphene[test]"`

Reply:
The version of pytest installed by graphene is may be not compatible with python 3.10. We need to make sure pytest>=6.2.5 is installed. Here's the pytest release note which introduced [py3.10 support](https://github.com/DataDog/dd-trace-py/pull/3646/commits/129f9b373d38fe20872e3d229e76d19d1fb06143). Or we could just test with python 3.9 and avoid this mess